mmv1/third_party/terraform/services/bigquery/resource_bigquery_table.go

```diff
@@ -502,7 +503,8 @@ func ResourceBigQueryTable() *schema.Resource {
 								json, _ := structure.NormalizeJsonString(v)
 								return json
 							},
-							Description: `A JSON schema for the external table. Schema is required for CSV and JSON formats and is disallowed for Google Cloud Bigtable, Cloud Datastore backups, and Avro formats when using external tables.`,
+							Description:   `A JSON schema for the external table. Schema is required for CSV and JSON formats and is disallowed for Google Cloud Bigtable, Cloud Datastore backups, and Avro formats when using external tables.`,
+							ConflictsWith: []string{"view", "materialized_view"},
```

Comment:
It looks like this is the wrong schema field - this is `external_data_configuration.schema`

Reply:
Thanks for the catch, fixed